NEWS

```diff
@@ -9,6 +9,7 @@ Current git version
   * Fix handling of delta -1 in 'focus_monitor' and 'cycle_monitor'
   * Fixed precision decimals in the layout tree (more reliable in- and output
     of fractions in frame splits)
+  * New optional argument 'split' for splitting non-leaf frames
```

Comment:
"New optional argument for the 'split' command" maybe?

Reply:
Yes; fixed!

Reply:
Thanks!